src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIPipelineService.java

```diff
@@ -128,4 +156,52 @@ private String localizedParamOptionLabel(Locale locale, String workflowName, Str
             return paramName + "." + optionName;
         }
     }
+
+    private List<SavedPipelineParameters> getSavedPipelineParameters(IridaWorkflow workflow, Locale locale) {
+        IridaWorkflowDescription description = workflow.getWorkflowDescription();
+        List<IridaWorkflowParameter> workflowParameters = description.getParameters();
+        String pipelineName = description.getName()
+                .toLowerCase();
+        List<SavedPipelineParameters> savedParameters = new ArrayList<>();
+
+        /*
+        If there are no parameters just return an empty list.
+         */
+        if (workflowParameters == null) {
+            return savedParameters;
+        }
+
+        /*
+        Get the default parameter set
+         */
+        List<PipelineParameter> defaultParameters = workflowParameters.stream()
+                .filter(p -> !p.isRequired())
+                .map(p -> new PipelineParameter(
+                        p.getName(),
+                        messageSource.getMessage("pipeline.parameters." + pipelineName + "." + p.getName(), new Object[]{}, locale),
+                        p.getDefaultValue()))
+                .collect(Collectors.toList());
+        savedParameters.add(new SavedPipelineParameters(0L, messageSource.getMessage("workflow.parameters.named.default", new Object[]{}, locale), defaultParameters));
```

Comment:
Shouldn't `0L` be the actual id. Wouldn't this just set the id to 0 for all of the SavedPipelineParameters?

Reply:
So this is a special case because it is the "default" set it doesn't actually have it own identifier.